ee/tables/execparsers/nixenv/parser.go

```diff
@@ -0,0 +1,28 @@
+//go:build darwin
+// +build darwin
+
+package nixenv
```

Comment:
This is a plain `NewMapXml` parser, right? Nothing nix about it? I would name it `mapxml` or something.  Maybe we'll be able to reuse it.